docusaurus.config.js

```diff
@@ -335,10 +335,6 @@ module.exports = {
             to: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/example-use-cases',
             from: '/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/about-rke1-templates/example-use-cases'
           },
-          {
-            to: '/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates',
-            from: '/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates'
-          },
```

Comment:
We probably should leave this block in for now until we decide on some guidelines on how we want to handle redirects in general. This redirect block and most of the ones surrounding it were added as part of the restructuring in https://github.com/rancher/rancher-docs/pull/234. 

Assuming it works then the full redirect chain (after combining with the one being added) would be

`/how-to-guides/advanced-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates` --> `/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-cluster-templates` --> `/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates`